src/main/scala/cromwell/engine/SymbolStore.scala

```diff
@@ -26,9 +26,46 @@ class SymbolStore(binding: WdlBinding, inputs: Map[FullyQualifiedName, WdlValue]
     assignSymbolStoreEntry(fullyQualifiedName, value, input = true)
   }
 
+  binding.workflows foreach {workflow =>
```

Comment:
Can run a for comp over this, something like:

    for (
    workflow <- binding.workflows;
    call <- workflow.calls;
    (k, v) <- call.inputMappings) yield assignSymbolStoreEntry(s"${call.fullyQualifiedName}.$k", v, input = true)